magda-web-client/src/Components/Search/SearchSuggestionBox.js

```diff
@@ -39,12 +39,25 @@ class SearchSuggestionBox extends Component {
             recentSearches: this.retrieveLocalData("recentSearches"),
             selectedItemIdx: null
         };
+        this.cacheImgs();
         this.createSearchDataFromProps(this.props);
         this.searchInputRef = null;
         this.onSearchInputKeyDown = this.onSearchInputKeyDown.bind(this);
         this.containerRef = null;
     }
 
+    cacheImg(img) {
```

Comment:
minor nitpick: could this live inside `cacheImgs()` like 

```javascript
const cacheImg = () => {
    const imgLoader = new Image();
    imgLoader.src = img;
    this.cacheImages.push(imgLoader);
}
```

Just a bit concerned that it depends on `cacheImgs()` being called first to initialize `this.cacheImages`, so it'd be really easy for some other developer to come and call it first by mistake.

Reply:
I think it's a good idea~ 
Will update it.
Thank you~